sdk/src/wallet/bindings/nodejs/examples/4a-send-amount.js

```diff
@@ -8,9 +8,6 @@ async function run() {
     try {
         const manager = await getUnlockedManager();
         const account = await manager.getAccount('Alice');
-        console.log('Account:', account);
-
-        await account.sync();
```

Comment:
Is this not needed?

Reply:
if you follow the steps, whatever is needed should be stored locally

Reply:
Sure, but I added sync calls intentionally to illustrate their usage.